ui/components/ui/metafox-logo/metafox-logo.component.js

```diff
@@ -11,12 +11,34 @@ export default class MetaFoxLogo extends PureComponent {
     ///: BEGIN:ONLY_INCLUDE_IN(build-flask)
     src: PropTypes.string,
     ///: END:ONLY_INCLUDE_IN
+    ///: BEGIN:ONLY_INCLUDE_IN(build-mmi)
+    custodyImgSrc: PropTypes.string,
+    isUnlocked: PropTypes.bool,
+    ///: END:ONLY_INCLUDE_IN
   };
 
   static defaultProps = {
     onClick: undefined,
   };
 
+  ///: BEGIN:ONLY_INCLUDE_IN(build-mmi)
+  renderCustodyIcon(iconProps, custodyImgSrc) {
+    return (
+      <div>
+        <img
+          {...iconProps}
+          src={custodyImgSrc}
+          className={classnames(
+            'app-header__custody-logo',
+            'app-header__custody-logo--icon',
+          )}
+          alt=""
+        />
+      </div>
```

Comment:
we can remove div wrapper to keep only img tag as there's only one DOM
```suggestion
        <img
          {...iconProps}
          src={custodyImgSrc}
          className={classnames(
            'app-header__custody-logo',
            'app-header__custody-logo--icon',
          )}
          alt=""
        />
```